### PR TITLE
add wazo-setupd API documentation

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: content/setupd/description.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# [`wazo-setupd`](https://github.com/wazo-platform/wazo-setupd)
+
+A micro service to initialize a [Wazo](http://wazo.community) Engine.
```
Comment (chartrandf) [inline content/setupd/description.md:3]: Better to use : https://wazo-platform.org

File: content/sections.yaml [hunk 2]
```diff
@@ -90,6 +90,14 @@
         color: purple
         repository: wazo-websocketd
 
+      setup:
+        title: Initial system setup
```
Comment (chartrandf) [inline content/sections.yaml:94]: Capital letters in title.

```suggestion
        title: Initial System Setup
```

File: content/setupd/description.md (hunk 1 above)
Comment (fblackburn1) [reply content/setupd/description.md:3]: Yep, copy/paste from wazo-setupd README :sweat_smile:

**Comment by fblackburn1** [comment]
> Do you think we should remove the overview link and `description.md`? There's not additional information and it feels empty.

Yes I try to use "overview: false" But the code wasn't implemented for platform section ... So I copy/paste the wazo-setupd README because I'm lazy man.
Honestly, I don't really care about overview section. All I want is the API to refer some guys on the forum.

**Comment by chartrandf** [comment]
> > Do you think we should remove the overview link and `description.md`? There's not additional information and it feels empty.
> 
> Yes I try to use "overview: false" But the code wasn't implemented for platform section ... So I copy/paste the wazo-setupd README because I'm lazy man.
> Honestly, I don't really care about overview section. All I want is the API to refer some guys on the forum.

Good point, added this logic to platform website. If it's good for you it's good for me, i'll approve it.

**Comment by fblackburn1** [comment]
Good for me, Thanks :+1:

**Pull request merged**